netlify/next-runtime
# docs: remove docs files and update links from repo docs to netlify docs

### Summary

This PR:

- Removes the documentation from the repo docs folder so that the [Next.js docs on Netlify](https://docs.netlify.com/integrations/frameworks/next-js/overview/) are the only documentation resource to maintain
- Updates links in the README to direct to the Netlify docs instead of the repo docs
- Updates links in v4 Release Notes to direct to the Netlify docs instead of the repo docs
- I did **not** delete the docs folder because it contains release notes; if preferred, I can delete the folder and move the release notes out

### Test plan

- Verify links in the README all route to the appropriate [Netlify docs](https://docs.netlify.com/integrations/frameworks/next-js/overview/)
- Check that docs from this repo have been deleted as that info has all migrated to the docs site

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal

Closes [this issue](https://github.com/netlify/docs/issues/2612) on the docs repo
[Docs Audit: Next.js Migration](https://www.notion.so/netlify/Ecosystem-Docs-Hub-2a06105f5ca348deb56a7d1268d3ada7)

![fennec-fox](https://user-images.githubusercontent.com/52582720/215220202-23417391-54c7-49ef-8b34-c54ef8491ee6.png)
